06-reactJs/components/input.js

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var _ = require('lodash');
+
+var Input = React.createClass({
+    
+    propTypes: {
+            defaultValue: React.PropTypes.string,
+            onChange: React.PropTypes.func,
+            value: React.PropTypes.string,
+            type: React.PropTypes.string
+    },
+
+    getInitialState: function () {
+        return {value=''}
+    },
+
+    render: function () {
+        return (
+            <input {...this.getProps} />
+        );
+    },
+
+    getProps: function () {
+        return {
+            type="text",
```

Comment:
Key/values on an object should be defined with a colon, not an equals sign. (``type: 'text'``). Also, avoid double quotes on JS files!